prove_plot.py

```diff
@@ -0,0 +1,6 @@
+import matplotlib
```

Comment:
by default we were failing to plot after installing dependencies, because we might not have had the correct python tk installed. this little script lets us easily test we can plot